src/main/scala/cromwell/engine/store/SymbolStore.scala

```diff
@@ -0,0 +1,86 @@
+package cromwell.engine.store
+
+import cromwell.binding._
+import cromwell.binding.types.WdlType
+import cromwell.binding.values.{WdlObject, WdlValue}
+import cromwell.engine.store.SymbolStore._
+
+import scala.util.{Failure, Success, Try}
+
+object SymbolStore {
+  case class SymbolStoreKey(scope: String, name: String, iteration: Option[Int], input: Boolean)
+
+  object CallInputWdlFunctions extends WdlFunctions {
+    def getFunction(name: String): WdlFunction = {
```

Comment:
I think this is "from" me but just because of moving. We shouldn't be using `getX` as method names. I was going to change this to `function` but perhaps `wdlFunction` would be more appropriate?